### ISSUE TITLE
hello, I got  a question about the code

### ISSUE THREAD

**Issue by pak24** [miranthajayatilake/image_captioning#1]
I run the file image_captioning_model_v2.py, at last , the model was successfully generatedand,  i want to use an image to test the model,so  i add some code in the bottom .

`from keras.models import load_model 
model = load_model('image_captioning_model_v2.h5')
#begin test image
image_path = '0.jpg'
img = image.load_img(image_path, target_size=(150,150))
img = image.img_to_array(img)
img = np.expand_dim(img, axis=0)
result = model.predict(img)
print ('the 0.jpg predict caption is: ', result)`

I run this code，but the result is many numbers such as 0.00402736, 0.0438112,0.00465147...... 
I do not understand those numbers, and i wonder how to make this numbers into real english words.

I really look forword to your answer , thank you very much!
